src/SFA.DAS.Roatp.CourseManagement.Web/Models/Standards/StandardDetailsViewModel.cs

```diff
@@ -12,11 +15,51 @@ public class StandardDetailsViewModel
         public string RegulatorName { get; set; }
         public string Version { get; set; }
         public string CourseDisplayName => $"{CourseName} (Level {Level})";
+
         public bool IsStandardRegulated => !string.IsNullOrEmpty(RegulatorName);
+
         public string StandardInfoUrl { get; set; }
         public string ContactUsPhoneNumber { get; set; }
         public string ContactUsEmail { get; set; }
         public string ContactUsPageUrl { get; set; }
+        public List<ProviderCourseLocationViewModel> ProviderCourseLocations { get; set; }
+
+        public List<ProviderCourseLocationViewModel> SubRegionCourseLocations { get; set; }
+       
+        public ProviderCourseLocationViewModel NationalCourseLocation { get; set; }
+
+        public List<string> Regions()
+        {
+            return SubRegionCourseLocations
+                .OrderBy(x => x.RegionName)
+                .Distinct().Select(region => region.RegionName).Distinct().ToList();
+        }
+
+
+        public string LocationSummary()
```

Comment:
Can be a get property

Reply:
I think SonarCloud reported it as a code smell when I did that, so I changed it to this.  I'll try it, and if SonarCloud complains, I'll put it back again

Reply:
Yes, this is now done